testing/trino-testing/src/main/java/io/trino/testing/TestingConnectorBehavior.java

```diff
@@ -58,6 +58,7 @@
     SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS(SUPPORTS_RENAME_TABLE),
 
     SUPPORTS_ADD_COLUMN,
+    SUPPORTS_ADD_COLUMN_WITH_COMMENT,
```

Comment:
Not necessarily. SUPPORTS_COMMENT_ON_COLUMN probably means setting a comment on existing column, which is different.

Just SUPPORTS_ADD_COLUMN since we want all these to be "as much defaults to true as reasonable"

Reply:
> There exist a series of connectors (e.g.: Postgres, MsSQL) which do support adding columns, but silently ignore the comment provided.

What are we supposed to do in this case if we don't have a new testing behaviour enum value? The check for the comment will obviously fail in the newly added `testAddColumnWithComment`.

Reply:
i think we agreed to have the new option (SUPPORTS_ADD_COLUMN_WITH_COMMENT)
it just should default to SUPPORTS_ADD_COLUMN
```
SUPPORTS_ADD_COLUMN_WITH_COMMENT(SUPPORTS_ADD_COLUMN),
```